week-04/solutions/divisible.rb

```diff
@@ -0,0 +1,56 @@
+range_input = (1..100)
```

Comment:
Nice job with giving different solutions but next time just give a single solution that you find the most efficient.